lib/playlistGenerator/ConcatSource.js

```diff
@@ -171,6 +171,19 @@ ConcatSource.prototype.checkExpires = function (clock,totalDuration) {
     return paths;
 };
 
+ConcatSource.prototype.removeClip = function() {
+
```

Comment:
removeClip is not an appropriate name since concat source has no notion of clips use chunk or smh